components/menu.module.css

```
@@ -1 +1,167 @@
-/* the style for the menu goes here */
+.hamburger {
+  position: fixed;
+  z-index: 1000;
+  top: 3.5rem;
+  left: 2.5rem;
+	width: 3rem;
+  height: 2.2rem;
+  cursor: pointer;
+}
+
+.hamburger_line {
+  position: absolute;
+  top: 0;
+  left: 0; 
+  width: 3rem;
+  height: 0.2rem;
+  background-color: var(--accent-color);
+}
+
+.hamburger_line::before {
+  content: '';
+  position: absolute;
+  top: 0.8rem;
+  width: 3rem;
+  height: 0.2rem;
+  background-color: var(--accent-color);
+}
+
+.hamburger_line::after {
+  content: '';
+  position: absolute;
+  top: 1.6rem;
+  width: 3rem;
+  height: 0.2rem;
+  background-color: var(--accent-color);
+}
+
+.cross_line {
+  position: absolute;
+  top: 0.5rem;
+  left: 0;
+  width: 3rem;
+  height: 0.2rem;
+  background-color: black;
+  transform: rotate(45deg);
+}
+
+.cross_line::after {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 3rem;
+  height: 0.2rem;
+  background-color: black;
+  transform: rotate(90deg);
+}
+
+.menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 70%;
+  height: 100vh;
+  background-color: var(--accent-color);
+  transition: all 0.5s ease-in-out;
+  z-index: 100;
+  padding: 8rem 5rem;
+}
+
+.menu_hidden {
+  position: fixed;
+  top: 0;
+  left: -100vw;
+  width: 70%;
+  height: 100vh;
+  background-color: var(--accent-color);
+  z-index: 100;
+  padding: 8rem 5rem;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu ul, .menu_hidden ul {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+}
+
+.menu li, .menu_hidden li {
+  font-family: var(--title-font);
+  font-size: 2rem;
+  text-decoration: none;
+  border: none;
+  list-style: none;
+}
+
+.menu li:hover, .menu_hidden li:hover {
+  color: white;
+  /* border-bottom: 1px solid black; */
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(7, 7, 7, 0.493);
+  backdrop-filter: blur(4px);
+  z-index: 98;
+  transition: all 0.5s ease-in-out;
+}
+
+.overlay_hidden {
+  display: hidden;
+}
+
+/*================ Laptop ==================*/
+  
+@media screen and (min-width: 1024px) {
+  .menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 400px;
+    height: 100vh;
+    background-color: rgb(233, 0, 0);
+    transition: all 0.5s ease-in-out;
+  }
+
+  .close_btn {
+    color: black;
+    margin: 20px;
+    font-size: 60px;
+    font-family: 'Times New Roman', Times, serif;
+  }
+
+  .menu ul {
+    position: absolute;
+  }
+
+  .menu li {
+    font-size: 25px;
+    margin: 15px 0;
+    padding: 10px 0;
+    text-decoration: none;
+    border: none;
+    list-style: none;
+  }
+
+  .menu li:hover {
+    color: white;
+    border-bottom: 1px solid black;
```

```suggestion
    border-bottom: 1px solid white;
```
Can we change it to white colour?